Documentation/ExtensionArchitecture/Extbase/Reference/UriArguments.rst

```diff
@@ -0,0 +1,63 @@
+.. include:: /Includes.rst.txt
+
+.. index:: Extbase; URI arguments; arguments
+
+===================================
+URI arguments and reserved keywords
+===================================
+
+Extbase uses special URI arguments to pass variables
+to Controller arguments and the framework itself.
+
+.. todo: Check for completion of this
+
+Extbase uses a prefixed URI argument scheme that relies
+on plugin configuration.
+
+For example, the example extension `EXT:blog_example` would use:
+
+..  code-block:: plaintext
+
+    https://example.com/blog/?tx_blogexample_bloglist[action]=show&tx_blogexample_bloglist[controller]=Post&tx_blogexample_bloglist[post]=4711&cHash=...
+
+    // Actually, the [] parameters are often URI encoded, so this is emitted:
+    https://example.com/blog/?tx_blogexample_bloglist%5Baction%5D=show&tx_blogexample_bloglist%5Bcontroller%5D=Post&tx_blogexample_bloglist%5Bpost%5D=4711&cHash=...
+
+as the created URI to execute the `showAction` of the Controller `PostController`
+within the plugin `BlogList`.
+
+The following arguments are evaluated:
+
+*   `tx_(extensionName)_(pluginName)[action]`: Controller action to execute
+*   `tx_(extensionName)_(pluginName)[controller]`: Controller containing the action
+*   `tx_(extensionName)_(pluginName)[format]`: Output format (usually HTML, can also be JSON or custom types)
+*   `cHash` - the cHash is always calculated to validate the URI is allowed to be called (see :ref:`chash`)
```

Comment:
```suggestion
*   `cHash` - the cHash is always calculated to validate that the URI can be called. (see :ref:`chash`)
```

Reply:
I think it is important to differentiate here that chash doesn't prevent a URI from being called, it just results in a "denied access" check. So I would like to keep the "allowed to" wording here, would that be ok with you?

Reply:
I only had a problem with the two `is` in the original sentence.
I don't have any problem to keep `is allowed to`

Reply:
Ah, I understand. I committed a slightly modified version now, how about that? :)